terraform/monitoring/config/alert.rules.yml

```diff
@@ -28,6 +28,17 @@ groups:
     labels:
       environment: production
       severity: high
+  - alert: ProdWorkerCPUHigh
+    expr: avg by ( app ) (cpu{app=~"teaching-vacancies-worker-production"}) > 75
+    for: 5m
+    annotations:
+      summary: Worker CPU usage high
+      description: "CPU usage at least 60% for more than 5 minutes (current value: {{ $value }})"
```

Comment:
Copypasta 🍝

Reply:
- Corrected description for split out alerts
- Updated documentation